### PR TITLE
fix(source-declarative-manifest): fix models

### PR DESCRIPTION
**Pull request by artem1205**
## What

Fix new protocol models

## How

use Serializer

## Review guide

1. `airbyte-integrations/connectors/source-declarative-manifest/source_declarative_manifest/run.py`

## Can this PR be safely reverted and rolled back?

- [ ] YES 💚
- [ ] NO ❌

### CODE REVIEW

**Review by alafanechere — APPROVED** [review]

**Comment by artem1205** [comment]
@alafanechere ,
CI fails on QA check: 
```
source-declarative-manifest - ❌ Failed - Connectors must have a changelog entry for each version: 
```

But I cannot find any docs for this connector. 
How can we merge this?

**Comment by alafanechere** [comment]
> @alafanechere , CI fails on QA check:
> 
> ```
> source-declarative-manifest - ❌ Failed - Connectors must have a changelog entry for each version: 
> ```
> 
> But I cannot find any docs for this connector. How can we merge this?

I think the changelog is here: https://github.com/airbytehq/airbyte/blob/master/docs/integrations/sources/low-code.md#L1

**Comment by artem1205** [comment]
@alafanechere,
found another comment in metadata:
```
# This version should not be updated manually - it is updated by the CDK release workflow.
```

What should we do?

File: docs/integrations/sources/low-code.md [hunk 1]
```diff
@@ -7,94 +7,95 @@ The changelog below is automatically updated by the `bump_version` command as pa
...
-| 0.80.0 | 2024-04-09 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.80.0 |
-| 0.79.2 | 2024-04-09 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.79.2 |
-| 0.79.1 | 2024-04-05 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.79.1 |
-| 0.79.0 | 2024-04-05 | [36501](https://github.com/airbytehq/airbyte/pull/36501) | Bump CDK version to 0.79.0 |
-| 0.78.9 | 2024-04-04 | [36834](https://github.com/airbytehq/airbyte/pull/36834) | Update CDK dependency to version 0.78.9 (before new publishing flow) |
+| Version | Date       | Pull Request                                                                                 | Subject                                                              |
+|:--------|:-----------|:---------------------------------------------------------------------------------------------|:---------------------------------------------------------------------|
+| 5.0.1   | 2024-09-02 | [45088](https://github.com/airbytehq/airbyte/pull/45088)                                     | Bump CDK version to 5.0.1                                            |
```
Comment (girarda) [inline docs/integrations/sources/low-code.md:12]: CDK version is bumped to 5.0.0.

File: airbyte-integrations/connectors/source-declarative-manifest/pyproject.toml [hunk 2]
```diff
@@ -3,7 +3,7 @@ requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
-version = "5.0.0"
+version = "5.0.1"
```
Comment (girarda) [inline airbyte-integrations/connectors/source-declarative-manifest/pyproject.toml:6]: I don't think 5.0.0 has been released so we should be good to keep the same version number

**Review by girarda — APPROVED** [review]

File: airbyte-integrations/connectors/source-declarative-manifest/pyproject.toml (hunk 2 above)
Comment (girarda) [reply airbyte-integrations/connectors/source-declarative-manifest/pyproject.toml:6]: edit: it's been released on pypi, but not docker :/

**Comment by alafanechere** [comment]
> @alafanechere, found another comment in metadata:
> 
> ```
> # This version should not be updated manually - it is updated by the CDK release workflow.
> ```
> 
> What should we do?

Oh yes. Let's revert the bump version changes. We can ignore failing QA checks related to version increment / changelog. I'll approve and merge + log an issue to handle this case in QA checks.

**Comment by octavia-approvington** [comment]
You did it! <br/> finish line

**Review by octavia-approvington — APPROVED** [review]

**Pull request merged**